src/tests/account/test_account.cairo

```diff
@@ -5,8 +5,7 @@ use openzeppelin::account::Account::PublicKeyImpl;
 use openzeppelin::account::Account;
 use openzeppelin::account::AccountABIDispatcher;
 use openzeppelin::account::AccountABIDispatcherTrait;
-use openzeppelin::account::QUERY_VERSION;
-use openzeppelin::account::TRANSACTION_VERSION;
+use openzeppelin::account::Account::{TRANSACTION_VERSION, QUERY_VERSION};
```

Comment:
i like this line particularly, it conveys what i cared about. in the future `account::EthAccount` could have its own and be equally clear by reading this.